docs/misc/changelog.md

```diff
@@ -0,0 +1,50 @@
+# Changelog
+
+Our releases are accompanied by changelogs based on the
+[Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format. We are using
+the [towncrier](https://github.com/twisted/towncrier) utility to generate them
+at the time a new version is released. There are currently four such changelogs
+for different components of the repository:
+
+* **[`core/CHANGELOG.md`](https://github.com/trezor/trezor-firmware/blob/master/core/CHANGELOG.md)** for Trezor T firmware
+* **[`legacy/firmware/CHANGELOG.md`](https://github.com/trezor/trezor-firmware/blob/master/legacy/firmware/CHANGELOG.md)** for Trezor 1 firmware
+* **[`legacy/bootloader/CHANGELOG.md`](https://github.com/trezor/trezor-firmware/blob/master/legacy/bootloader/CHANGELOG.md)** for Trezor 1 bootloader
+* **[`python/CHANGELOG.md`](https://github.com/trezor/trezor-firmware/blob/master/python/CHANGELOG.md)** for Python client library
+
+## Adding changelog entry
+
+[`towncrier`](https://github.com/twisted/towncrier) aims to create changelogs
+that are convenient to read, at the expense of being somewhat inconvenient to
+create. Furthermore every changelog entry has to be linked to a GitHub issue or
+pull request number. If you don't want to create an issue just to satisfy this
+rule you can use self-reference to your change's pull request number by first
+creating the PR and then adding the entry.
+
+There are a few types of changelog entries, as described by the [Keep a
+Changelog](https://keepachangelog.com/en/1.0.0/) format:
+
+* `added`
+* `changed`
+* `deprecated`
+* `removed`
+* `fixed`
+* `security`
+* `incompatible`, used only by `python`
+
+Entries are added by creating files in the `.changelog.d` directory where the
```

Comment:
can't we use the `towncrier` tool to do this? Including instructions for that would be preferable.

Reply:
Added mention in 5b80e81f4887f153875e7e37d4d81490cf414c12.